### ISSUE TITLE
COW importer fails to download on MacOS

### ISSUE THREAD

**Issue by cjyetman** [vincentarelbundock/countryyear#8]
[COW importer fails to download a CSV file](https://github.com/vincentarelbundock/countryyear/blob/master/utilities/correlates_of_war.R#L16) on MacOS, likely because the URL is redirected and base's `download.file` apparently handles that differently on different platforms.

using `method = 'curl'` works on my machine, but I don't know the consequences of that on other platforms...
```r
download.file(url, tmp, method = 'curl')
```
alternatively, using the `curl` package's drop-in replacement `curl_download` is apparently more portable...
```r
curl::curl_download(url, tmp)
```

I could change it myself obviously, but giving @vincentarelbundock the opportunity to decide which path to take.

**Comment by vincentarelbundock** [comment]
No difference for me. Better if it works on more machines, I suppose.

**Comment by cjyetman** [comment]
resolved by [commit 3de440d](https://github.com/vincentarelbundock/countryyear/commit/3de440ddc9a4607d3da23b97ed29d7366be5326a)
